holoviews/util.py

```diff
@@ -69,15 +72,17 @@ def _dynamic_operation(self, map_obj):
             def dynamic_operation(*key, **kwargs):
                 self.p.kwargs.update(kwargs)
                 return self._process(map_obj[key], key)
-            return dynamic_operation
-
-        def dynamic_operation(*key, **kwargs):
-            key = key[0] if map_obj.mode == 'open' else key
-            self.p.kwargs.update(kwargs)
-            _, el = util.get_dynamic_item(map_obj, map_obj.kdims, key)
-            return self._process(el, key)
-
-        return dynamic_operation
+        else:
+            def dynamic_operation(*key, **kwargs):
+                key = key[0] if map_obj.mode == 'open' else key
+                self.p.kwargs.update(kwargs)
+                _, el = util.get_dynamic_item(map_obj, map_obj.kdims, key)
+                return self._process(el, key)
+        if isinstance(self.p.operation, ElementOperation):
```

Comment:
I'm a little confused. ``_dynamic_operation`` seems to be called once [here](https://github.com/ioam/holoviews/blob/ca945bf92e41bcfe9d3975bd53fbdbf4719f01a3/holoviews/util.py#L36) and I don't see the ``operation`` parameter then being inspected/used on the ``callback`` variable anywhere in ``Dynamic``. I am yet to spot where the ``operation`` parameter of ``OperationCallable`` is used...so I'm assuming I've missed it.